python/server/auto_generated/qgsconfigcache.sip.in

```diff
@@ -24,6 +25,11 @@ Cache for server configuration.
 %End
   public:
 
+    static void initialize( QgsServerSettings *settings );
+%Docstring
+Initialize the strategy
```

Comment:
Is the term "strategy" defined somewhere in this API doc? It's used twice on this class' doc and impossible to understand without context.

Same for the term `entry` further down.

Reply:
> Is the term "strategy" defined somewhere in this API doc

This is no longer referenced in the sip from the doc since the method is now out of python scope. Strategy definition is described in  `QgsSettings`.

> Same for the term entry further down.

This what the term used in the original doc, what term should we use ?